secrets-provider-test-chart/templates/job.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Values.global.namespace }}
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ .Values.rbac.serviceAccount.name }}
+      containers:
+      - image: {{ .Values.secretsProvider.image }}
+        imagePullPolicy: Never
+        name: {{ .Values.secretsProvider.name }}
+        env:
+        - name: SECRETS_DESTINATION
+          value: {{ .Values.secretsProviderEnvironment.secretsDestination | quote }}
+
+        # TODO, remove requirement in code
+        - name: CONTAINER_MODE
+          value: init
+
+        - name: MY_POD_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.name
+
+        - name: MY_POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+
+        - name: CONJUR_APPLIANCE_URL
+          value:  {{ required "conjurApplianceUrl is a required configuration" .Values.secretsProviderEnvironment.conjurApplianceUrl | quote }}
```

Comment:
Remove all the `required`.
We will do the validation using `JsonSchema` in other task.

Reply:
disgree @abrahamko